infra/eligibility_screener/template/ecs.tf

```diff
@@ -0,0 +1,102 @@
+# security group for screener
+resource "aws_security_group" "allow-screener-traffic" {
+  name = "allow_screener_traffic"
+  description = "This rule blocks all traffic unless it is HTTPS for the eligibility screener"
+  vpc_id = "vpc-032e680f92b88bb68" # don't like that this is hardcoded; default vpc
+  # may need to update application to accept 80 then redirect.
+
+  ingress {
+    description = "TCP traffic from VPC"
+    from_port = 80
+    to_port   = 80
+    protocol = "tcp"
+    cidr_blocks = ["0.0.0.0/8"]
+  }
+
+  egress {
+    description = "allow all outbound traffic from screener"
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+}
+
+resource "aws_ecr_repository" "eligibility-screener-repository" {
+  name                 = "eligibility-screener-repo"
+  image_tag_mutability = "IMMUTABLE"
+}
```

Comment:
Do we want to add encryption at rest to this repo?

Reply:
lol, disregard. I answered my own question: Every image you push to ECR is already encrypted by default using an industry-standard AES-256 encryption algorithm. 

https://aws.amazon.com/about-aws/whats-new/2020/07/ecr-supports-encryption-images-aws-kms-keys/#:~:text=Every%20image%20you%20push%20to,it%20protects%20data%20at%20rest.